src/sERC20.sol

```diff
@@ -31,4 +33,8 @@ contract sERC20 is ERC20 {
 
         _burn(owner, amount);
     }
+
```

Comment:
nitpick: `spender` could be `operator` in function `burn`

Reply:
fixed